lib/mix/lib/mix/tasks/release.init.ex

```diff
@@ -301,6 +301,7 @@ defmodule Mix.Tasks.Release.Init do
     setlocal enabledelayedexpansion
 
     pushd .
+    %~d0
     cd "%~dp0\.."
     set RELEASE_ROOT=%cd%
     popd
```

Comment:
I have been reading the docs and it seems this would also work:

```
pushd "%~dp0\.."
set RELEASE_ROOT=%cd%
popd
```

Because, opposite to `cd`, `pushd` changes the path. can you please give it a try too? :heart:

Reply:
Yes, it works well too.
And it seems little bit cleaner than playing with old "dos" command.

I'll do another pull request